panda-whale-vekm/dinehard
# Mayson/front end features

Description
This pull request provides minor styling and a fix to the font issue. Font should now appear for everyone.

Asana Task
N/A

Testing Instructions
run npm start to spin up the server and observe styling changes.

Additional Notes, Images, etc.
